trace-mini-agent/src/trace_processor.rs

```diff
@@ -41,12 +41,12 @@ impl TraceChunkProcessor for ChunkProcessor {
     fn process(&mut self, chunk: &mut pb::TraceChunk, root_span_index: usize) {
         trace_utils::set_serverless_root_span_tags(
             &mut chunk.spans[root_span_index],
-            self.config.function_name.clone(),
+            self.config.app_name.clone(),
             &self.config.env_type,
         );
         for span in chunk.spans.iter_mut() {
             trace_utils::enrich_span_with_mini_agent_metadata(span, &self.mini_agent_metadata);
-            trace_utils::enrich_span_with_azure_metadata(span);
+            trace_utils::enrich_span_with_azure_function_metadata(span);
```

Comment:
I'm a little confused why were are changing everything to app name but still using function metadata? Is this because springs apps is built on top of functions?

Reply:
Originally the mini agent was written just for functions. So the name of the traced service was always a function, hence the property `function_name` in the mini agent config.

Now that Azure Spring Apps, which are unrelated to functions, are also traced with the mini agent I wanted to rename this property to something more generic. I figured `app_name` was generic enough and could represent either a function or a spring app. Let me know if you have another idea for this property name!

In the code you highlighted I just renamed the method to be clear it's only being used for Azure Functions since the old name, `enrich_span_with_azure_metadata` sounded like it could be used for Azure Spring Apps (and it is not used for those).